app/workers/notifications/new_reaction_worker.rb

```diff
@@ -0,0 +1,19 @@
+module Notifications
+  class NewReactionWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :medium_priority, retry: 10
+
+    def perform(reaction_data, receiver_data, service = Notifications::Reactions::Send)
```

Comment:
you can remove the service as a parameter and mock `Notifications::Reactions::Send` in the worker specs directly

Reply:
I am sorry I didn't get you here.
I have already used `let(:reaction_service) { double }` to make fake reaction_service.

Reply:
What I mean is that you don't need really need to pass the service as an argument and you can test the real one in the specs. I'llve leave you some suggestions

```suggestion
    def perform(reaction_data, receiver_data)
```